robottelo/helpers.py

```diff
@@ -280,6 +281,20 @@ def add_remote_execution_ssh_key(hostname, key_path=None, **kwargs):
     ssh.add_authorized_key(server_key, hostname=hostname, **kwargs)
 
 
+def add_remote_execution_capsule_to_subnet(hostname, capsule_id=1):
```

Comment:
Honestly, by name add_remote_execution_capsule_to_subnet I would understand a method that accepts subnet and capsule as parameters. This is a little misleading as it actually takes a host, and parses a subnet assigned to it.

This makes the method too specific.